server/gokbg3/grails-app/controllers/org/gokb/rest/RefdataController.groovy

```diff
@@ -154,4 +154,34 @@ class RefdataController {
     result.data = resultData
     render result as JSON
   }
+
+  @Secured(['IS_AUTHENTICATED_ANONYMOUSLY'])
+  def coverageDepth() {
+    def result = [:]
+    def resultData = []
+    def cat = null
+    def base = grailsApplication.config.serverURL + '/'+namespace
+
+    cat = RefdataCategory.findByLabel("TitleInstancePackagePlatform.CoverageDepth")
```

Comment:
Since coverage info has been moved completely to TIPPCoverageStatement, we should use `TIPPCoverageStatement.CoverageDepth` here.

Reply:
Not sure how to get a list of possible coverage refdata values from that TIPPCoverageStatement...

Reply:
Since the domain class TIPPCoverageStatement has its own `coverageDepth`, it also has a separate RefdataCategory `TIPPCoverageStatement.CoverageDepth` (which contains the same values as `TitleInstancePackagePlatform.CoverageDepth`). Since we return the Database-ID of the values as well and support updating the property by providing an ID, we should take care to output the correct values here. Therefore it should be 

`cat = RefdataCategory.findByLabel("TIPPCoverageStatement.CoverageDepth")`

Reply:
ok. done